src/06-serial-communication/nix-tooling.md

```diff
@@ -19,42 +19,9 @@ the board; check the next section, which contains instructions for older
 revisions. If you do have a newer revision skip the next section and move to the
 "minicom" section.
 
-## Older revisions of the discovery board / external serial module
+## minicom
 
-Connect the serial module to your computer and let's find out what name the OS assigned to it.
-
-> **NOTE** On macs, the USB device will named like this: `/dev/cu.usbserial-*`. You won't
-> find it using `dmesg`, instead use `ls -l /dev | grep cu.usb` and adjust the following 
-> commands accordingly!
-
-``` console
-$ dmesg | grep -i tty
-(..)
-[  +0.000155] usb 3-2: FTDI USB Serial Device converter now attached to ttyUSB0
-```
-
-But what's this `ttyUSB0` thing? It's a file of course! Everything is a file in \*nix:
-
-``` console
-$ ls -l /dev/ttyUSB0
-crw-rw-rw- 1 root uucp 188, 0 Oct 27 00:00 /dev/ttyUSB0
-```
-
-> **NOTE** if the permissions above is `crw-rw----`, the udev rules have not been set correctly
-> see [udev rules](../03-setup/linux.html#udev-rules)
```

Comment:
Was this already explained somewhere else?

Reply:
This is again only for outdated revision of the board/the serial adapter, on my linux I can just call minicom onto the device with my permissions looking like what is described as "wrong" here:
```
λ ls -l /dev/ttyACM0
crw-rw----. 1 root plugdev 166, 0 Jan 21 11:56 /dev/ttyACM0
```
So I guess this is fine? At least as long as you are part of the plugdev group of course but that might also be a Fedora specific thing, not quite sure :/

Reply:
I did not have to set up permissions either in ubuntu so I would agree to the removal.